NEWS.md

```diff
@@ -1,3 +1,10 @@
+# scales 1.0.0.9000
+
+## Minor bug fixes and improvements
```

Comment:
This subheading is unnecessary when listing changes to the dev version, it's added when organizing changes just before release. Thanks for your patience with all these little nits -- you caught us right at release weeks so versions in NEWS and DESCRIPTION, haven't been bumped yet.

Reply:
No worries.  I'd be making similar comments on people submitting PRs to my repositories.